ParallelRoadTool/NetTypeItem.cs

```diff
@@ -15,4 +15,24 @@ public NetTypeItem(NetInfo netInfo, float horizontalOffset, float verticalOffset
             IsReversed = isReversed;
         }
     }
+
+    //helper class needed for serialization because NetInfo class can not be serialized
+    public class PresetNetItem
+    {
+        public float HorizontalOffset;
+        public bool IsReversed;
+        public string NetName;
+        public float VerticalOffset;
+
+        public PresetNetItem(string netName, float horizontalOffset, float verticalOffset, bool isReversed)
+        {
+            NetName = netName;
```

Comment:
Do you have the chance to change your game's language and see if you can still load a preset saved while using English? I'm not 100% sure but I think that we can't use network's name as it depends on game's language.

Reply:
Tested with German. Seems to work just fine.  Names were saved in English to XML even if language was set to German.
English saved preset, loaded in German: OK
German saved preset, loaded in English: OK

Same preset loaded in English and in German:
![sl english](https://user-images.githubusercontent.com/25418803/43366924-8bab65b6-934d-11e8-9a9f-361112cd3fbb.PNG)

![sl german](https://user-images.githubusercontent.com/25418803/43366925-9532b7b0-934d-11e8-87dd-4f56571814ac.PNG)